### ISSUE TITLE
Cannot reproduce results in paper.

### ISSUE THREAD

**Issue by alexlfyang** [guanlinchao/bert-dst#4]
I run the code on WOZ2.0 dataset and cannot get the results of 87.7% reported in the paper. The used hyper-parameters are the default and I got the result of around 77.8% +- 2%. 
Is there any other trick?

**Comment by guanlinchao** [comment]
With this hyperparameter, I trained 300-400 epochs on WOZ 2.0, and used the validation set's joint goal accuracy to determine the best epoch.
For DSTC2, I trained 50 epochs.
